students/steve-long/lesson02-basic-functions/ex-2-2-fizzBuzz/fizzBuzz.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+# ========================================================================================
+# Python210 | Fall 2020
+# ----------------------------------------------------------------------------------------
+# Lesson02
+# Fizz Buzz Exercise (fizzBuzz.py)
+# Steve Long 2020-09-19 | v0
+#
+# Requirements:
+# -------------
+# Write a program that prints the numbers from 1 to 100 inclusive.
+# But for multiples of three print "Fizz" instead of the number.
+# For the multiples of five print "Buzz" instead of the number.
+# For numbers which are multiples of both three and five print "FizzBuzz" instead.
+#
+# Implementation:
+# ---------------
+#	fizzBuzz([<start>[,<end>]])
+#
+# Script Usage:
+# -------------
+# 	python fizzBuzz.py [<start> [<end>]]
+#		<start> ::= Starting number (see function fizzBuzz for details.)
+#		<end>   ::= Ending number (see function fizzBuzz for details.)
+#
+# History:
+# --------
+# 000/2020-09-20/sal/Created.
+# ========================================================================================
+
+import sys
+	
+def fizzBuzz(start = 1, end = 100):
+	"""
+	fizzBuzz([<start>[,<end>]])
+	--------------------------------------------------------------------------------------
+	For non-negative int values from <start> to <end>, print the following:
+	* "Fizz" for multiples of 3
+	* "Buzz" for multiples of 5
+	* "FizzBuzz" for multiples of 3 and 5
+	* The number for all other values
+	
+	Entry: <start> ::= Starting int value (default is 1).
+	       <end>   ::= Ending int value (default is 100).
+	Exit:  Returns True.
+	"""
+	for n in range(start, (end + 1)):
+		s = ("FizzBuzz" if (((n % 3) == 0) and ((n % 5) == 0)) else \
+		("Fizz" if ((n % 3) == 0) else \
+		("Buzz" if ((n % 5) == 0) else n)))
+		print(s)
```

Comment:
per python best practices, you should be using exactly 4 spaces (no tabs)

note that it might look like 4 spaces in your editor if it's set for tabs - but as you can see on github you are using 8 spaces for indentation.